src/auro-checkbox.scss

```diff
@@ -3,13 +3,28 @@
 
 // ---------------------------------------------------------------------
 
+/* stylelint-disable custom-property-empty-line-before,
+   custom-property-pattern,
+   selector-max-class,
+   scss/selector-nest-combinators,
+   order/properties-order,
+   max-nesting-depth,
+   selector-max-combinators,
+   selector-max-compound-selectors,
+   selector-class-pattern,
+   rule-empty-line-before,
+   scss/selector-no-redundant-nesting-selector,
+   no-descending-specificity,
+   no-duplicate-selectors,
+   declaration-empty-line-before */
+
 // Import Auro tokens
 @import "./../node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables";
 @import "./../node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties";
```

Comment:
This is no longer needed since we are not post-processing any of the CSS custom properties.